src/NUnitFramework/framework/Constraints/Comparers/StreamsComparer.cs

```diff
@@ -39,8 +42,15 @@ public static EqualMethodResult Equal(object x, object y, ref Tolerance toleranc
                     return EqualMethodResult.ComparedEqual;
             }
 
+#if NETFRAMEWORK
```

Comment:
request:

Can we remove the need for `#if NETFRAMEWORK` here? `ArrayPool` is available for all of NUnit's build targets so there's no need to make this .NET Core-specific :)

Reply:
The reason I added it is because it gives an error on .net framework for [IDE0005 ](https://learn.microsoft.com/en-gb/dotnet/fundamentals/code-analysis/style-rules/ide0005?pivots=lang-csharp-vb) because we are not using it

Reply:
Why not use it on .NET framework too? It should be available.

Reply:
Aah I was not sure if this was a .NET Core only feature! I've added it across the frameworks now